learning_journal/models.py

```diff
@@ -40,8 +41,28 @@ def all(cls, session=None):
 
     @classmethod
     def by_id(cls, requested_id, session=None):
+        """Returns a single entry, given an id."""
         if session is None:
             session = DBSession
         return session.query(cls).get(requested_id)
 
 # Index('my_index', Entry.title, unique=True, mysql_length=255)
+
+
+class User(Base):
+    __tablename__ = 'users'
+    id = Column(Integer, primary_key=True)
+    username = Column(Unicode(length=255), nullable=False, unique=True, index=True)
+    password = Column(Unicode, nullable=False)
+
+    @classmethod
+    def get_user(cls, username, session=None):
+        if session is None:
+            session = DBSession
+        return [o.user for o in session.query(cls).filter(cls.user == username)]
```

Comment:
We want the one and only user returned or `None`, not a list of users.  You want to do something here similar to what you did with the `by_id` method of our Entry model.